docs/admin/maintenance/ossec_alerts.rst

```diff
@@ -49,36 +54,23 @@ The SMTP relay that you use should support SASL authentication and SMTP
 TLS protocols TLSv1.2, TLSv1.1, and TLSv1. Most enterprise email
 solutions should be able to meet those requirements.
 
-Below are the values you must specify to configure OSSEC correctly.
-For first-time installs, you can use the
-:ref:`configuration playbook<configure_securedrop>`, or edit
-``install_files/ansible-base/group_vars/all/site-specific`` manually.
-
-- *OSSEC Alert Public Key*:
-  ``ossec_alert_gpg_public_key``
-- Fingerprint of key used when encrypting OSSEC alerts:
-  ``ossec_gpg_fpr``
-- The email address that will receive alerts from OSSEC:
-  ``ossec_alert_email``
-- The reachable hostname of your SMTP relay: ``smtp_relay``
-- The secure SMTP port of your SMTP relay: ``smtp_relay_port``
-  (typically 25, 587, or 465. must support TLS encryption)
-- Email username to authenticate to the SMTP relay: ``sasl_username``
-- Domain name of the email used to send OSSEC alerts: ``sasl_domain``
-- Password of the email used to send OSSEC alerts: ``sasl_password``
+These values must be set in the 
+:ref:`configuration playbook<configure_securedrop>` by running the
+``securedrop-admin sdconfig`` command, which will prompt for each of the
+items listed above.
```

Comment:
It might be good to add a reminder here that this only updates the configuration, but does not apply it to the servers yet.